examples/chart/teleport-demo/docker/build-all.sh

```diff
@@ -1,15 +1,25 @@
-#!/usr/bin/env bash
+#!/bin/bash
 set -e
+ONE=$1
+TWO=$2
 VERSION=3.2.0
 if [[ "$1" != "" ]]; then
     VERSION=$1
     shift
 fi
-set -e
+GCPROJECT=kubeadm-167321
+if [[ "" != "${TWO}" ]]; then
+    GCPROJECT=${TWO}
+    shift
+fi
```

Comment:
The comparison here is backwards - you want to make sure that the variable is not empty. The actual effect here will be the same as you're checking against an empty string, but convention/best practice is to put your comparison after the variable that you're checking.

You also don't need to set `$TWO` because you're just going to assign that to `$GCPROJECT` anyway.

```bash
if [[ "$2" != "" ]]; then
    GCPROJECT=$2
    shift
fi
```